src/BeforeValidationInterface.php

```diff
@@ -11,7 +11,7 @@
  */
 interface BeforeValidationInterface
 {
-    public function shouldSkipOnEmpty(): bool;
+    public function shouldSkipOnEmpty($value): bool;
```

Comment:
Could it be typed as bool|callable?

Reply:
Nope. It's not `skipOnEmpty` parameter's value (as it was initially suggested, having different types), it's a validated value. Changed to `mixed` and improved naming to avoid confusion.